Add option to static analyzer inputs (#20391)
DataDog/documentation: content/en/continuous_integration/static_analysis/_index.md

Provide the following inputs:

| Name           | Description                                                                                                                | Required | Default         |
|----------------|----------------------------------------------------------------------------------------------------------------------------|----------|-----------------|
| `service`      | The name of the service to tag the results with.                                                                           | Yes      |                 |
| `env`          | The environment to tag the results with. `ci` is a helpful value for this input.                                           | No       | `none`          |
| `cpu_count`    | Set the number of CPUs used by the analyzer. Defaults to the number of CPUs available.                                     | No       |                 |

> | `subdirectory` | A subdirectory path to run an analysis in. The path is relative to the root directory of your repository.                  | No       |                 |

Reviewer: ```suggestion
| `subdirectory` | The subdirectory path the analysis should be limited to. The path is relative to the root directory of the repository.                  | No       |                 |
```
